lib/core/lib/ops/teamOps.js

```diff
@@ -510,6 +510,57 @@ function buildTeamOps (db, config) {
         cb()
       })
     },
+    /**
+     * List a nested team in an organization
+     *
+     * @param  {Object}   params { organizationId, id, limit, page } where page is 1-indexed
+     * @param  {Function} cb
+     */
+    listNestedTeams: function listNestedTeams (params, cb) {
+      let { organizationId, id, limit, page } = params
+
+      Joi.validate({ organizationId, id, page, limit }, validationRules.listNestedTeams, function (err) {
+        if (err) return cb(Boom.badRequest(err))
+
+        let sqlQuery = SQL`
+          WITH total AS (
+            SELECT COUNT(*) AS cnt
+            FROM teams
+            WHERE org_id = ${organizationId} AND team_parent_id = ${id}
+          )
+          SELECT
+            teams.id,
+            teams.name,
+            teams.description,
+            teams.team_parent_id as parent_id,
+            teams.path,
+            teams.org_id,
+            t.cnt::INTEGER AS total,
+            COUNT(team_members.team_id) AS users_count
+          FROM teams
+          LEFT JOIN team_members ON team_members.team_id = teams.id
+          INNER JOIN total AS t ON 1=1
+          WHERE org_id = ${organizationId} AND team_parent_id = ${id}
+          GROUP BY teams.id, teams.name, teams.description, teams.team_parent_id, teams.path, teams.org_id, t.cnt
+          ORDER BY UPPER(name);
+        `
+
+        if (limit) {
+          sqlQuery.append(SQL` LIMIT ${limit}`)
+        }
+        if (limit && page) {
+          let offset = (page - 1) * limit
+          sqlQuery.append(SQL` OFFSET ${offset}`)
+        }
+
+        db.query(sqlQuery, function (err, result) {
+          if (err) return cb(Boom.badImplementation(err))
+
```

Comment:
I tested manually with volume data and the results coming back perfect I just noticed if I go outside the range of IDs that it returns an empty array, which is fine but a team that exists could have an empty array so it might give the user the impression that a team exists that doesn't unless we throw back a 400?

@jimmymintzer @dberesford @dberesford  just wondering here should we put in 400 'team not found with id' here if the team does not exist... unfortunately 0 results isn't the same thing... maybe a call to get team first returning it's error if it has one, otherwise this?